src/server/game/Quests/QuestDef.h

```diff
@@ -388,6 +388,9 @@ class TC_GAME_API Quest
         uint32 _startItemCount        = 0;
         uint32 _rewardMailSenderEntry = 0;
         uint32 _specialFlags          = 0; // custom flags, not sniffed/WDB
+
+        // Helpers
+        uint32 RoundXPValue(uint32 xp) const;
```

Comment:
Which would you prefer?  I'm thinking static having to choose between the two, since it's relevant only to the scope of quest.

Also, is this a general pattern adherence used in this code base?  That is, not to have private methods for internal class methods?  Or is it due to the nature of the purpose of the Quest class (abstraction).  If so, I'll keep that in mind from now on when I need class subroutines.